ui/apps/platform/src/Containers/Dashboard/PatternFly/Widgets/AgingImages.tsx

```diff
@@ -0,0 +1,258 @@
+    });
+    const timeRangeCounts = data ?? previousData;
+
+    const inputError = timeRangeTupleIndices.some((index) => !isNumberInRange(timeRanges, index))
+        ? new Error('Invalid image ages')
+        : undefined;
+
+    return (
+        <WidgetCard
+            isLoading={loading && !timeRangeCounts}
+            error={error || inputError}
+            errorTitle={inputError && 'Incorrect image age values'}
+            errorMessage={
+                inputError &&
+                'There was an error retrieving data. Image ages must be in ascending order.'
+            }
+            header={
+                <Flex direction={{ default: 'row' }}>
+                    <FlexItem grow={{ default: 'grow' }}>
+                        <Title headingLevel="h2">
+                            {getWidgetTitle(searchFilter, timeRanges, timeRangeCounts)}
+                        </Title>
+                    </FlexItem>
+                    <FlexItem>
+                        <Dropdown
+                            className="pf-u-mr-sm"
+                            toggle={
+                                <DropdownToggle
+                                    id={`${fieldIdPrefix}-options-toggle`}
+                                    toggleVariant="secondary"
+                                    onToggle={toggleOptionsOpen}
+                                >
+                                    Options
+                                </DropdownToggle>
+                            }
+                            position="right"
+                            isOpen={isOptionsOpen}
+                        >
+                            <Form className="pf-u-px-md pf-u-py-sm">
+                                <FormGroup
+                                    fieldId={`${fieldIdPrefix}-time-range-0`}
+                                    label="Image age values"
+                                >
+                                    {timeRangeTupleIndices.map((index) => (
+                                        <div key={index}>
+                                            <Checkbox
+                                                aria-label="Toggle image time range"
+                                                id={`${fieldIdPrefix}-time-range-${index}`}
+                                                name={`${fieldIdPrefix}-time-range-${index}`}
+                                                className="pf-u-mb-sm pf-u-display-flex pf-u-align-items-center"
+                                                isChecked={timeRanges[index].enabled}
+                                                onChange={() => dispatch({ type: 'toggle', index })}
+                                                label={
+                                                    <TextInput
+                                                        aria-label="Image age in days"
+                                                        style={{ minWidth: '100px' }}
```

Comment:
The checkboxes and number inputs alone are small enough to cause the menu label to wrap. This ensures we have a large enough minimum size to not break the layout of the dropdown.